test/integration/server.h

```diff
@@ -40,6 +40,7 @@ struct FieldValidationConfig {
   bool allow_unknown_static_fields = false;
   bool reject_unknown_dynamic_fields = false;
   bool ignore_unknown_dynamic_fields = false;
+  bool suppress_deprecated_warning_logs = false;
```

Comment:
I don't think this is being used, and can probably be removed (the call in server.cc can be with `false`).

Reply:
removed the `suppress_deprecated_warning_logs` variable and directly updated the call in server.cc with `false`.